src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java

```diff
@@ -0,0 +1,91 @@
+package io.lbry.browser.reactmodules;
+
+import android.app.Activity;
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.app.NotificationManagerCompat;
+import android.support.v4.content.ContextCompat;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+
+import io.lbry.browser.MainActivity;
+import io.lbry.browser.R;
+
+public class BackgroundMediaModule extends ReactContextBaseJavaModule {
+
+    private static final int NOTIFICATION_ID = 900;
+
+    private static final String NOTIFICATION_CHANNEL_ID = "io.lbry.browser.MEDIA_PLAYER_NOTIFICATION_CHANNEL";
+
+    public static final String ACTION_PLAY = "io.lbry.browser.ACTION_MEDIA_PLAY";
+
+    public static final String ACTION_PAUSE = "io.lbry.browser.ACTION_MEDIA_PAUSE";
+
+    public static final String ACTION_STOP = "io.lbry.browser.ACTION_MEDIA_STOP";
+
+    private boolean channelCreated;
+
+    private Context context;
+
+    public BackgroundMediaModule(ReactApplicationContext reactContext) {
+        super(reactContext);
+        this.context = reactContext;
+    }
+
+    @Override
+    public String getName() {
+        return "BackgroundMedia";
+    }
+
+    @ReactMethod
+    public void showPlaybackNotification(String title, String publisher, String uri, boolean paused) {
+        if (!channelCreated && Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            NotificationManager notificationManager =
+                (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+            NotificationChannel channel = new NotificationChannel(
+                NOTIFICATION_CHANNEL_ID, "LBRY Media", NotificationManager.IMPORTANCE_LOW);
+            channel.setDescription("LBRY media player");
```

Comment:
Does this render within the notification?
Maybe we should adjust this text a bit.

Reply:
I'll need to play with this a bit on my device

Reply:
No, it doesn't. Notification channels are usually configured in notification settings on Android 8.0 and higher. Basically, you can turn channels on or off which lets you determine the notifications related to channels that you want to see. https://developer.android.com/training/notify-user/channels